### PR TITLE
NMEA mode setting for UARTs

### PR DESCRIPTION
**Pull request by fnoble**
cc: @gsmcmullin @jacobmcnamee

### CODE REVIEW

**Comment by anonymous-travis-swiftnav** [comment]
## v0.2-8-g3f3aa39
+ [Artifacts (HITL Dashboard Page)](http://hitl-dashboard.swiftnav.com/files/swiftnav-artifacts-pull-requests/piksi_buildroot/v0.2-8-g3f3aa39/)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v0.2-8-g3f3aa39/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v0.2-8-g3f3aa39

**Comment by anonymous-travis-swiftnav** [comment]
## v0.3.0-9-g0645141
+ [Artifacts (HITL Dashboard Page)](http://hitl-dashboard.swiftnav.com/files/swiftnav-artifacts-pull-requests/piksi_buildroot/v0.3.0-9-g0645141/)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v0.3.0-9-g0645141/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v0.3.0-9-g0645141

File: board/piksiv3/rootfs-overlay/etc/monitrc [hunk 1]
```diff
@@ -27,17 +27,17 @@ check process zmq_adapter_rpmsg_piksi100 with pidfile /var/run/zmq_adapter_rpmsg
     start program = "/etc/init.d/S83zmq_adapter_rpmsg_piksi100 start"
     stop program = "/etc/init.d/S83zmq_adapter_rpmsg_piksi100 stop"
 
-check process zmq_adapter_tcp_listen with pidfile /var/run/zmq_adapter_tcp_listen.pid
-    start program = "/etc/init.d/S83zmq_adapter_tcp_listen start"
-    stop program = "/etc/init.d/S83zmq_adapter_tcp_listen stop"
+check process zmq_adapter_rpmsg_piksi100 with pidfile /var/run/zmq_adapter_rpmsg_piksi101.pid
```
Comment (jacobmcnamee) [inline board/piksiv3/rootfs-overlay/etc/monitrc]: I think you combined `zmq_adapter_rpmsg_piksi100` and `zmq_adapter_rpmsg_piksi101`

File: package/piksi_system_daemon/src/main.c [hunk 2]
```diff
@@ -50,6 +51,97 @@ bool baudrate_notify(struct setting *s, const char *val)
   return ret;
 }
 
+static const char const * port_mode_enum[] = {"SBP", "NMEA", NULL};
+static struct setting_type port_mode_settings_type;
+static int TYPE_PORT_MODE = 0;
+enum {PORT_MODE_SBP, PORT_MODE_NMEA};
+u8 uart0_mode = PORT_MODE_SBP;
```
Comment (jacobmcnamee) [inline package/piksi_system_daemon/src/main.c]: `static`

**Comment by jacobmcnamee** [comment]
Added a couple comments, otherwise lgtm

**Comment by anonymous-travis-swiftnav** [comment]
## v0.3.0-30-gc78df99
+ [Artifacts (HITL Dashboard Page)](http://hitl-dashboard.swiftnav.com/files/swiftnav-artifacts-pull-requests/piksi_buildroot/v0.3.0-30-gc78df99/)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v0.3.0-30-gc78df99/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v0.3.0-30-gc78df99

**Pull request merged**